ui/pages/Project/components/FamilyTable/IndividualRow.jsx

```diff
@@ -131,13 +130,6 @@ CaseReviewStatus.propTypes = {
 }
 
 const SHOW_DATA_MODAL_CONFIG = [
-  {
```

Comment:
why are we moving this link out of the `SHOW_DATA_MODAL_CONFIG`? It seems that since there is no scenario where we treat `hasRnaOutlierData` different from `hasRnaSpliceOutlierData` it makes sense to only have `hasRnaOutlierData` and change the server side logic to factor in both data types

Reply:
The link is moved from the `SHOW_DATA_MODAL_CONFIG ` because we will open a new page, not a modal, for both expression and splicing outliers. We combine has-data indicators for these outliers into one `hasRnaOutlierData`.